src/Symfony/Component/Workflow/SupportStrategy/WorkflowSupportStrategyInterface.php

```diff
@@ -18,5 +18,5 @@
  */
 interface WorkflowSupportStrategyInterface
 {
-    public function supports(WorkflowInterface $workflow, object $subject): bool;
+    public function supports(WorkflowInterface $workflow, $subject): bool;
```

Comment:
I don't know if it's a BC break / cc @fabpot or @nicolas-grekas

Reply:
It is a BC break, so it's not allowed.

Reply:
I don't think I can do what I want without all these BC Breaks :/

So, shall we move it to Symfony6 or it's not worth it ?

Reply:
We never break BC without an upgrade path. So, Symfony 6 won’t help.

Reply:
This PR brings a good change in my opinion. Providing an upgrade path for those kind of changes was easier in the bad old days without typing.
Is there another good or even better name for the interface to deprecated this one?

Reply:
I have a stupid question that I can't answer.  
I'm sorry to bother you with this.

How loosening the typing makes it a BC break ?

I tried this code :
```
<?php
class A {
	function f($arg) {
		var_dump($arg);
	}
}

class B extends A {
	function f(object $arg) {
		var_dump($arg);
	}
}

$a = new A();
$a->f((new \stdClass));

$a = new A();
$a->f(1);

$b = new B();
$b->f((new \stdClass));
```

But everything is working as expected.  
What am I missing ?